lms/djangoapps/teams/models.py

```diff
@@ -319,25 +319,20 @@ def get_memberships(cls, username=None, course_ids=None, team_ids=None):
         return queryset
 
     @classmethod
-    def user_in_team_for_course(cls, user, course_id, topic_id=None):
+    def user_in_team_for_teamset(cls, user, course_id, topic_id=None):
```

Comment:
If we're doing this, would we want to change the argument name to `teamset_id` and make it nonoptional?
Are we using it anywhere in the old way? Because we probably shouldn't be.

Reply:
Updated argument name and removed default value.  Also, updated docstring to match.